### PR TITLE
[stable/kong] revamp and simplify readme

### PR DESCRIPTION
**Pull request by hbagdi**
#### What this PR does / why we need it:

Revamps and re-write major portions of the README

#### Special notes for your reviewer:

Please read through the new README itself. The diff is nearly unreadable.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/kong/FAQs.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Frequently Asked Questions (FAQs)
+
```
Comment (rainest) [inline stable/kong/FAQs.md:2]: Was additional content not yet committed here? It seems odd to add this with just the header.

File: stable/kong/README.md [hunk 2]
```diff
@@ -215,82 +346,67 @@ To use Kong Enterprise, change your image to a Kong Enterprise image and set
...
-kind: Secret
-metadata:
-  creationTimestamp: "2019-05-17T21:45:16Z"
-  name: kong-enterprise-license
-  namespace: default
-  resourceVersion: "48695485"
-  selfLink: /api/v1/namespaces/default/secrets/kong-enterprise-license
-  uid: 0f2e8903-78ed-11e9-b1a6-42010a8a02ec
-type: Opaque
+ype: Opaque
```
Comment (rainest) [inline stable/kong/README.md]: Extra line not fully caught in the deleted content above.

**Review by rainest — CHANGES_REQUESTED** [review]

File: stable/kong/README.md (hunk 2 above)
Comment (hbagdi) [reply stable/kong/README.md]: Fixed

File: stable/kong/FAQs.md (hunk 1 above)
Comment (hbagdi) [reply stable/kong/FAQs.md:2]: Yes, not ideal. Adding real soon but not in this PR. There is a humongous PR coming with loads of improvements.

File: stable/kong/README.md [hunk 3]
```diff
@@ -1,66 +1,190 @@
...
+  can be controlled via `admin.enabled` and `env.admin_listen` parameters.
+
+## Configuration
+
+### Kong parameters
 
 | Parameter                          | Description                                                                           | Default             |
 | ---------------------------------- | ------------------------------------------------------------------------------------- | ------------------- |
 | image.repository                   | Kong image                                                                            | `kong`              |
+| image.repository                   | Kong image                                                                            | `kong`              |
```
Comment (shashiranjan84) [inline stable/kong/README.md]: duplicate

Comment (hbagdi) [reply stable/kong/README.md]: Fixed,

**Review by shashiranjan84 — APPROVED** [review]
/lgtm

**Pull request merged**